apps/prairielearn/src/ee/pages/institutionAdminLti13/institutionAdminLti13.html.ts

```diff
@@ -339,6 +339,13 @@ ${JSON.stringify(instance.custom_fields, null, 3)}</textarea
       </form>
 
       <hr />
+      <p>
+        For testing, have the LMS admin configure their OpenID Connect Initiation Url to
```

Comment:
```suggestion
        For testing, have the LMS admin configure their OpenID Connect Initiation URL to
```

Reply:
Changed in 517b8a1 but that capitalization was based on the Canvas configuration page:
![image](https://github.com/PrairieLearn/PrairieLearn/assets/10765199/a48cc3d9-dab2-473c-89da-760717a66453)